src/Plutarch/Extra/ScriptContext.hs

```diff
@@ -147,7 +149,7 @@ pvalueSpent = phoistAcyclic $
 
     @since 1.1.0
 -}
-pisTokenSpent :: forall (s :: S). Term s (PAssetClass :--> PBuiltinList PTxInInfo :--> PBool)
+pisTokenSpent :: forall (tag :: Symbol) (s :: S). Term s (PAssetClass tag :--> PBuiltinList PTxInInfo :--> PBool)
```

Comment:
API change, so needs a `since` bump. Also a changelog entry if it doesn't already have one.